svc-bie-kafka/src/main/java/gov/va/vro/services/bie/service/AmqpTopicSender.java

```diff
@@ -0,0 +1,6 @@
+package gov.va.vro.services.bie.service;
+
+public interface AmqpTopicSender {
```

Comment:
If the `Topic` in the name refers to the exchange type, aren't you using a `fanout` exchange type? Since there's also a `topic` exchange type, this interface name may be confusing.